README.md

```diff
@@ -80,6 +136,16 @@ delta_sharing.load_as_pandas(table_url)
 delta_sharing.load_as_spark(table_url)
 ```
 
+If the table supports history sharing, the connector can query table changes.
```

Comment:
Is there more reference material on what is history sharing?

Reply:
It's different fields in tableConfig between databricks and oss server, added an explaination.